pages/common/nano.md

```diff
@@ -7,7 +7,7 @@
 
 `nano {{filename}}`
 
-- Start nano in terminal with {filename} at line {line}, column {column}:
+- Start nano in terminal with filename at line,column:
```

Comment:
Perhaps `at specified line and column` will read better?

Reply:
I agree with @zdroid's suggestion.